### DISCUSSION TITLE
Please I need your help sir.

### DISCUSSION THREAD

**Discussion by Dikachi101** [community/community#19880] [Other Feature Feedback, Questions, & Ideas]
My name is Michael from Nigerian and I am a beginner programmer but I am having issue with my KivyMD project.
 I have been working on this project as part of my learning but after some configurations I did, my window started showing me a screen that is turned 90 degree to the right with some error messages:
FBO initialization not completed:

**Comment by YouthguyZ** [comment]
be honstly, i don't understand you, maybe you can make it clear😥

**Comment by pedrosantosara** [comment]
give example in image

**Comment by GathsaraH** [comment]
Link your Github repo and please explain more about your quetion.

**Comment by COLLiDER4D** [comment]
Need more info
